cicecore/shared/ice_calendar.F90

```diff
@@ -126,10 +126,9 @@ module ice_calendar
          dumpfreq               ! restart frequency, 'y','m','d'
 
       character (len=char_len), public :: &
-         dumpfreq_base,    & ! restart frequency basetime ('zero', 'init')
-         histfreq_base,    & ! history frequency basetime ('zero', 'init')
-         calendar_type       ! differentiates Gregorian from other calendars
-                             ! default = ' '
+         dumpfreq_base = 'zero', & ! restart frequency basetime ('zero', 'init')
+         histfreq_base = 'init', & ! history frequency basetime ('zero', 'init')
+         calendar_type             ! define calendar type
```

Comment:
why are the default values set here in **`ice_calendar`** instead of in **`ice_init`** as for other namelist flags ?

Reply:
They are also set in ice_init in the standard method.  The reason that I set them here as well is to help the unit testing work better.  calchk doesn't go thru any CICE initialization, so having defaults set here means I don't have to explicitly set them when I run the calchk unit test.  One could argue whether that is the best way to do it.  I also made a similar change to ndtd in ice_calendar.